app/recordtransfer/templates/recordtransfer/signupform.html

```diff
@@ -31,7 +25,7 @@
                     <label for="{{ field.id_for_label }}">{{ field.label }}</label>
                     {{ field }}
                     {% if field.help_text %}
-                        <div tooltip-content="{{ field.help_text }}" class="help-tooltip help-icon">
+                        <div tooltip-content={{ field.help_text }} class="help-tooltip help-icon">
```

Comment:
Removing the double quotes here has broken the tooltips because `field.help_text` may have spaces in it. See for example this tooltip which only says "This":

![Screenshot 2025-05-13 082112](https://github.com/user-attachments/assets/04ec8ab8-e37b-45f9-aac4-b5101c39b0bf)

It should say what's in the `help_text` here: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/blob/c6cb705cdda41c025289c21b9461bc32d699d30e/app/recordtransfer/forms/__init__.py#L42

The fact that that form is in `__init__.py` is not great, I can open a new issue to address that.

```suggestion
                        <div tooltip-content="{{ field.help_text }}" class="help-tooltip help-icon">
```

Reply:
Resolved by https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/561/commits/ef9bab4acdce0cedfffc595fb938ff2a6b469f87